config/config.go

```diff
@@ -195,6 +195,7 @@ type Config struct {
 	KubernetesBackendTrafficAlgorithm                    kubernetes.BackendTrafficAlgorithm `yaml:"-"`
 	KubernetesDefaultLoadBalancerAlgorithm               string                             `yaml:"kubernetes-default-lb-algorithm"`
 	KubernetesForceService                               bool                               `yaml:"kubernetes-force-service"`
+	KubernetesNoPoll                                     bool                               `yaml:"kubernetes-no-poll"`
```

Comment:
Maybe better an enum to create a DisableDataClient or something like that.
I think we should not be as specific if we introduce a new configuration.

Reply:
That's also a nice idea, thanks!